sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,28 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Does enabling/using the feature will result in any new API calls?
+    Describe them with their impact keeping in mind the [supported limits][]
+    (e.g. 5000 nodes per cluster, 100 pods/s churn) focusing mostly on:
+     - components listing and/or watching resources they didn't before
+     - API calls that may be triggered by changes of some Kubernetes
+       resources (e.g. update object X based on changes of object Y)
+     - periodic API calls to reconcile state (e.g. periodic fetching state,
+       heartbeats, leader election, etc.)
+  - Does enabling/using the feature will result in supporting new API
+    objects? How many of those will be supported (and how that translates
+    to limitations for users)?
+  - Does enabling/using the feature will result in increasing size or count
+    of the existing API objects?
+  - Does enabling/using the feature will result in increasing time taken
+    by any operations covered by [existing SLIs/SLOs][] (e.g. by adding
```

Comment:
What about removing the "covered by existing SLIs/SLOs" part to adhere to the "avoid specif questions" rule?
I think that our SLIs/SLOs list is unfortunately not exhaustive, there are things like scheduler throughput that is not covered by any scalability SLI/SLO but is an important factor in widely understood scalability.

IMHO, we should ask people to, basing on their gut feeling", say whether a change will add some extra work / increase some operation time and then rely on the KEP reviewers / approvers to ask clarifying questions and, if it still doesn't make the case clear, loop in sig-scalability folks.

Reply:
I think I disagree - the whole point of SLIs/SLOs is that they should cover what users really care about. So it's fine to increase latency of things. Obviously we're not in the point where we would like to be, but relying on "gut feeling" is not a good option in my opinion. 
Regarding "scheduling throughput", we should finally invest into make that somewhat official. I would like to avoid asking question: "mention everything that you think may be slower". Explicit reference to those things is actually better imho.